.ci/pipeline_definitions

```diff
@@ -67,9 +79,3 @@ vpn2:
             internal_scp_workspace:
               channel_name: 'C9CEBQPGE' #sap-tech-gardener
               slack_cfg_name: 'scp_workspace'
-        component_descriptor:
-          component_name: 'github.com/gardener/vpn2'
```

Comment:
Why did you remove the name? Is it no longer required or is it just set implicitly?

Reply:
@ScheererJ : technically, component_name defaults to github-repository-url, so adding it will not have any effect (as long as the repository is not moved/renamed). If you prefer, I can add the `component_name` attribute again, of course.

Reply:
No, we can leave it as is for now.